lib/ble.toit

```diff
@@ -93,13 +100,15 @@ class Advertiser:
   The data is advertised for the given $duration and once every $interval.
     If the $duration is null, then the data is advertised indefinitely.
 
+  The advertise will include the given connection_mode, use one of the BLE_CONNECTION_MODE_* constants
```

Comment:
Break the long line and indent the continuation with 2 spaces like done in line 101.